ports/espressif/mpconfigport.mk

```diff
@@ -68,10 +65,14 @@ CIRCUITPY_USB = 0
 else ifeq ($(IDF_TARGET),esp32s2)
 # Modules
 CIRCUITPY_BLEIO = 0
+CIRCUITPY_ESPULP = 1
+CIRCUITPY_MEMORYMAP = 1
 
 else ifeq ($(IDF_TARGET),esp32s3)
 # Modules
 CIRCUITPY_PARALLELDISPLAY = 0
+CIRCUITPY_ESPULP = 1
+CIRCUITPY_MEMORYMAP = 1
 endif
```

Comment:
I refactored `mpconfigport.mk` recently and set the following convention:
```py
# Module on by default and conditionally turned off

CIRCUITPY_ESPULP ?= 1

ifeq ($(IDF_TARGET),esp32)
CIRCUITPY_ESPULP = 0
CIRCUITPY_MEMORYMAP = 0

else ifeq ($(IDF_TARGET),esp32c3)
CIRCUITPY_ESPULP = 0
CIRCUITPY_MEMORYMAP = 0
```

Reply:
I'd prefer to do the opposite so that future chips have it off by default.